py4j-python/src/py4j/tests/java_callback_test.py

```diff
@@ -318,6 +321,13 @@ def testProxyReturnerVoid(self):
         output = example.computeNothing(returner)
         self.assertEqual(output, 1)
 
+    def testProxyReturnerNull(self):
+        sleep()
+        example = self.gateway.jvm.py4j.examples.ReturnerExample()
+        returner = Returner()
+        output = example.computeNull(returner)
+        self.assertEqual(output, None)
```

Comment:
I was about to suggest to use assertIsNone, but it doesn't exist in Python 2.6, so assertEqual it is!